rust/rpc/src/lib.rs

```diff
@@ -339,7 +339,8 @@ impl<W: Write + Send> RpcLoop<W> {
                     }
                 }
             }
-        });
+        }).unwrap();
```

Comment:
If I understand correctly this will panic on an Err? But the next thing in the file is an if that returns either Ok or Err so panicing here doesn't really make sense at a quick glance.

Reply:
point